includes/class-wc-payments-fraud-service.php

```diff
@@ -110,7 +110,11 @@ private function prepare_sift_config( $config ) {
 		if ( $this->check_if_user_just_logged_in() ) {
 			$config['session_id'] = $this->get_cookie_session_id();
 		} else {
-			$config['session_id'] = $wpcom_blog_id . '_' . WC()->session->get_customer_id();
+			if ( is_a( WC()->session, 'WC_Session' ) && method_exists( WC()->session, 'get_customer_id' ) ) {
```

Comment:
nit: since `get_customer_id()` is already defined in `WC_Session`, `method_exists()` check is not necessary, but it's OK to be there. Hence, nit.

Reply:
You're right.